### ISSUE TITLE
3.0.0a1 - Line indented incorrectly, false positive

### ISSUE THREAD

**Issue by brainfoolong** [squizlabs/PHP_CodeSniffer#1133]
Hi, i don't know why but i'm getting those errors, sparsely. I think that is a bug because i cannot imagine that the suggested way is the correct way. Correct me or clarify me on if i miss something :)

Ruleset: PSR2 - Unchanged
Testfile: [Test.class.php.zip](https://github.com/squizlabs/PHP_CodeSniffer/files/442096/Test.class.php.zip)

As i said, those error's does not appear everytime.

![error](https://cloud.githubusercontent.com/assets/1684236/18045990/00406e0c-6dd6-11e6-94e7-c38bdbd01cde.png)

**Comment by aik099** [comment]
This is because of scope (function/class) indentation. I guess you've enabled sniff, that wants class in a file to indented X nesting levels (e.g. 0). Then each line in problematic method, that opens new scope (the `if` in above example) would complain about incorrect indentation.

**Comment by gsherwood** [comment]
Any chance you could post the actual code instead of a screenshot?

**Comment by brainfoolong** [comment]
@aik099 I use PHPCS with default PSR2 ruleset. I've not changed anything.
@gsherwood Well, i've broken it down to the smallest possible file.
[Test.class.php.zip](https://github.com/squizlabs/PHP_CodeSniffer/files/442096/Test.class.php.zip)

**Comment by gsherwood** [comment]
Thanks for posting that. I can confirm the issue exists in the current releases but has been fixed already. It was originally reported as bug #1101 (slightly different case, but same result). The next set of releases (hopefully this week) will have this fix.

When running the PSR2 standard over your code, I now get this:
```
FILE: temp.php
----------------------------------------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
----------------------------------------------------------------------------------------------------
  6 | ERROR | [ ] Each class must be in a namespace of at least one level (a top-level vendor name)
 31 | ERROR | [x] Line indented incorrectly; expected at least 20 spaces, found 19
----------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------------------------------------
```
So the only indent error is where you've got 3 spaces for an indent instead of 4, on line 31.

**Comment by brainfoolong** [comment]
@gsherwood Great, did not seen this issue before. It's not that big of a problem for me, i just use alpha software to help other dev's with reportings. I know that there may exist problems. Keep up the good work.

**Comment by gsherwood** [comment]
> i just use alpha software to help other dev's with reportings

Thanks a lot. I really appreciate the testing of the pre-release versions.
